### ISSUE TITLE
Css is clashing with the app's css 

### ISSUE THREAD

**Issue by ajayexpert** [jinkin1995/vue-json-edit#46]

**Comment by sylvainpolletvillard** [comment]
Indeed, this dependency is injecting Pure CSS library v1.0.0 globally at the document level, which is likely to break the styles of the applications. This is a must fix

**Comment by MerlinR** [comment]
Well, im having that issue, and seeing as it's been 2 years, imma just move on.
For clarification, for me it's affecting the <i> and inputs, such as buttons, dropdown's, etc, i cannot find a way to really stop it other then adding a class to all buttons.
